lib/session/sessionPlugin.js

```diff
@@ -119,7 +120,7 @@ const register = function (server, options) {
                 // If we used any additional auth headers when authenticating, we need to store them in the session
```

Comment:
could we update this comments, since we no longer store additionalAuthHeaders anymore

correct me if i'm wrong, we are looking for ["security_impersonate_as"] from additionalAuthHeaders, in case we didn't find any match, session.additionalAuthHeaders will set to "".

Reply:
It will be an empty object {}. filterAuthHeaders internally uses lodash https://lodash.com/docs/4.17.15#pick